[2021-02-25 06:09:21,409] INFO [GroupMetadataManager brokerId=0] Removed 0 expired offsets in 1 milliseconds. (kafka.coordinator.group.GroupMetadataManager)
[2021-02-25 06:19:21,474] INFO [GroupMetadataManager brokerId=0] Removed 0 expired offsets in 1 milliseconds. (kafka.coordinator.group.GroupMetadataManager)
[2021-02-25 06:29:21,459] INFO [GroupMetadataManager brokerId=0] Removed 0 expired offsets in 0 milliseconds. (kafka.coordinator.group.GroupMetadataManager)
[2021-02-25 06:42:47,175] INFO [GroupMetadataManager brokerId=0] Removed 0 expired offsets in 4 milliseconds. (kafka.coordinator.group.GroupMetadataManager)
[2021-02-25 06:46:48,265] INFO [ProducerStateManager partition=__consumer_offsets-23] Writing producer snapshot at offset 5779582 (kafka.log.ProducerStateManager)
[2021-02-25 06:46:48,271] INFO [Log partition=__consumer_offsets-23, dir=/tmp/kafka-logs] Rolled new log segment at offset 5779582 in 24 ms. (kafka.log.Log)
[2021-02-25 06:46:58,815] INFO [Log partition=__consumer_offsets-23, dir=/tmp/kafka-logs] Scheduling segments for deletion LogSegment(baseOffset=0, size=1084, lastModifiedTime=1614248108652, largestTime=1614245681999) (kafka.log.Log)
[2021-02-25 06:46:58,815] INFO [Log partition=__consumer_offsets-23, dir=/tmp/kafka-logs] Scheduling segments for deletion LogSegment(baseOffset=4128282, size=127, lastModifiedTime=1614252488022, largestTime=1614252488022) (kafka.log.Log)
[2021-02-25 06:47:03,957] INFO [Log partition=__consumer_offsets-23, dir=/tmp/kafka-logs] Scheduling segments for deletion LogSegment(baseOffset=4953932, size=104857550, lastModifiedTime=1614257208246, largestTime=1614257208246) (kafka.log.Log)
[2021-02-25 06:47:58,818] INFO [Log partition=__consumer_offsets-23, dir=/tmp/kafka-logs] Deleting segments LogSegment(baseOffset=0, size=1084, lastModifiedTime=1614248108652, largestTime=1614245681999) (kafka.log.Log)
[2021-02-25 06:47:58,818] INFO [Log partition=__consumer_offsets-23, dir=/tmp/kafka-logs] Deleting segments LogSegment(baseOffset=4128282, size=127, lastModifiedTime=1614252488022, largestTime=1614252488022) (kafka.log.Log)
[2021-02-25 06:47:58,821] INFO Deleted log /tmp/kafka-logs/__consumer_offsets-23/00000000000000000000.log.deleted. (kafka.log.LogSegment)
[2021-02-25 06:47:58,821] INFO Deleted log /tmp/kafka-logs/__consumer_offsets-23/00000000000004128282.log.deleted. (kafka.log.LogSegment)
[2021-02-25 06:47:58,821] INFO Deleted offset index /tmp/kafka-logs/__consumer_offsets-23/00000000000004128282.index.deleted. (kafka.log.LogSegment)
[2021-02-25 06:47:58,822] INFO Deleted time index /tmp/kafka-logs/__consumer_offsets-23/00000000000004128282.timeindex.deleted. (kafka.log.LogSegment)
[2021-02-25 06:47:58,822] INFO Deleted offset index /tmp/kafka-logs/__consumer_offsets-23/00000000000000000000.index.deleted. (kafka.log.LogSegment)
[2021-02-25 06:47:58,822] INFO Deleted time index /tmp/kafka-logs/__consumer_offsets-23/00000000000000000000.timeindex.deleted. (kafka.log.LogSegment)
[2021-02-25 06:48:03,959] INFO [Log partition=__consumer_offsets-23, dir=/tmp/kafka-logs] Deleting segments LogSegment(baseOffset=4953932, size=104857550, lastModifiedTime=1614257208246, largestTime=1614257208246) (kafka.log.Log)
[2021-02-25 06:48:03,964] INFO Deleted log /tmp/kafka-logs/__consumer_offsets-23/00000000000004953932.log.deleted. (kafka.log.LogSegment)
[2021-02-25 06:48:03,964] INFO Deleted offset index /tmp/kafka-logs/__consumer_offsets-23/00000000000004953932.index.deleted. (kafka.log.LogSegment)
[2021-02-25 06:48:03,964] INFO Deleted time index /tmp/kafka-logs/__consumer_offsets-23/00000000000004953932.timeindex.deleted. (kafka.log.LogSegment)
[2021-02-25 06:52:47,182] INFO [GroupMetadataManager brokerId=0] Removed 0 expired offsets in 0 milliseconds. (kafka.coordinator.group.GroupMetadataManager)
[2021-02-25 06:59:04,315] INFO Terminating process due to signal SIGHUP (org.apache.kafka.common.utils.LoggingSignalHandler)
[2021-02-25 06:59:04,394] INFO [KafkaServer id=0] shutting down (kafka.server.KafkaServer)
[2021-02-25 06:59:04,403] INFO [KafkaServer id=0] Starting controlled shutdown (kafka.server.KafkaServer)
[2021-02-25 06:59:04,718] INFO Socket error occurred: localhost/[0:0:0:0:0:0:0:1]:2181: Connection reset (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:04,950] INFO [ZooKeeperClient Kafka server] Waiting until connected. (kafka.zookeeper.ZooKeeperClient)
[2021-02-25 06:59:04,950] INFO [ZooKeeperClient Kafka server] Connected. (kafka.zookeeper.ZooKeeperClient)
[2021-02-25 06:59:06,792] INFO Opening socket connection to server localhost/[0:0:0:0:0:0:0:1]:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:06,796] INFO Socket error occurred: localhost/[0:0:0:0:0:0:0:1]:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:06,904] INFO [ZooKeeperClient Kafka server] Waiting until connected. (kafka.zookeeper.ZooKeeperClient)
[2021-02-25 06:59:08,205] INFO Opening socket connection to server localhost/[0:0:0:0:0:0:0:1]:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:08,208] INFO Socket error occurred: localhost/[0:0:0:0:0:0:0:1]:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:09,963] INFO Opening socket connection to server localhost/[0:0:0:0:0:0:0:1]:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:09,965] INFO Socket error occurred: localhost/[0:0:0:0:0:0:0:1]:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:11,224] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:11,225] INFO Socket error occurred: localhost/127.0.0.1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:13,239] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:13,240] INFO Socket error occurred: localhost/127.0.0.1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:14,993] INFO Opening socket connection to server localhost/[0:0:0:0:0:0:0:1]:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:14,994] INFO Socket error occurred: localhost/[0:0:0:0:0:0:0:1]:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:17,068] INFO Opening socket connection to server localhost/[0:0:0:0:0:0:0:1]:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:17,070] INFO Socket error occurred: localhost/[0:0:0:0:0:0:0:1]:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:18,929] INFO Opening socket connection to server localhost/[0:0:0:0:0:0:0:1]:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:18,931] INFO Socket error occurred: localhost/[0:0:0:0:0:0:0:1]:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:20,066] INFO Opening socket connection to server localhost/[0:0:0:0:0:0:0:1]:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:20,068] INFO Socket error occurred: localhost/[0:0:0:0:0:0:0:1]:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:21,462] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:21,462] INFO Socket error occurred: localhost/127.0.0.1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:23,378] INFO Opening socket connection to server localhost/[0:0:0:0:0:0:0:1]:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:23,379] INFO Socket error occurred: localhost/[0:0:0:0:0:0:0:1]:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:24,561] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:24,562] INFO Socket error occurred: localhost/127.0.0.1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:25,713] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:25,715] INFO Socket error occurred: localhost/127.0.0.1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:27,347] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:27,347] INFO Socket error occurred: localhost/127.0.0.1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:28,629] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:28,630] INFO Socket error occurred: localhost/127.0.0.1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:29,870] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:29,870] INFO Socket error occurred: localhost/127.0.0.1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:31,563] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:31,564] INFO Socket error occurred: localhost/127.0.0.1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:31,794] INFO [GroupCoordinator 0]: Member MicroLib-fcb21140-f2f0-4954-8cc1-157eec7acaa8 in group fedmonserv has failed, removing it from the group (kafka.coordinator.group.GroupCoordinator)
[2021-02-25 06:59:31,796] INFO [GroupCoordinator 0]: Preparing to rebalance group fedmonserv in state PreparingRebalance with old generation 1 (__consumer_offsets-27) (reason: removing member MicroLib-fcb21140-f2f0-4954-8cc1-157eec7acaa8 on heartbeat expiration) (kafka.coordinator.group.GroupCoordinator)
[2021-02-25 06:59:31,799] INFO [GroupCoordinator 0]: Group fedmonserv with generation 2 is now empty (__consumer_offsets-27) (kafka.coordinator.group.GroupCoordinator)
[2021-02-25 06:59:33,559] INFO Opening socket connection to server localhost/[0:0:0:0:0:0:0:1]:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:33,559] INFO Socket error occurred: localhost/[0:0:0:0:0:0:0:1]:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:35,536] INFO Opening socket connection to server localhost/[0:0:0:0:0:0:0:1]:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:35,537] INFO Socket error occurred: localhost/[0:0:0:0:0:0:0:1]:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:37,513] INFO Opening socket connection to server localhost/[0:0:0:0:0:0:0:1]:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:37,514] INFO Socket error occurred: localhost/[0:0:0:0:0:0:0:1]:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:39,501] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:39,501] INFO Socket error occurred: localhost/127.0.0.1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:40,867] INFO Opening socket connection to server localhost/[0:0:0:0:0:0:0:1]:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:40,869] INFO Socket error occurred: localhost/[0:0:0:0:0:0:0:1]:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:42,528] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:42,529] INFO Socket error occurred: localhost/127.0.0.1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:44,421] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:44,422] INFO Socket error occurred: localhost/127.0.0.1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:45,535] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:45,537] INFO Socket error occurred: localhost/127.0.0.1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:46,813] INFO Opening socket connection to server localhost/[0:0:0:0:0:0:0:1]:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:46,814] INFO Socket error occurred: localhost/[0:0:0:0:0:0:0:1]:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:48,717] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:48,718] INFO Socket error occurred: localhost/127.0.0.1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:50,116] INFO Opening socket connection to server localhost/[0:0:0:0:0:0:0:1]:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:50,117] INFO Socket error occurred: localhost/[0:0:0:0:0:0:0:1]:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:51,271] INFO Opening socket connection to server localhost/[0:0:0:0:0:0:0:1]:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:51,271] INFO Socket error occurred: localhost/[0:0:0:0:0:0:0:1]:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:52,624] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:52,627] INFO Socket error occurred: localhost/127.0.0.1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:53,973] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:53,975] INFO Socket error occurred: localhost/127.0.0.1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:55,413] INFO Opening socket connection to server localhost/[0:0:0:0:0:0:0:1]:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:55,413] INFO Socket error occurred: localhost/[0:0:0:0:0:0:0:1]:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:57,268] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:57,269] INFO Socket error occurred: localhost/127.0.0.1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:59,276] INFO Opening socket connection to server localhost/[0:0:0:0:0:0:0:1]:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2021-02-25 06:59:59,277] INFO Socket error occurred: localhost/[0:0:0:0:0:0:0:1]:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
